.github/workflows/sdk-publish.yaml

```
@@ -78,7 +82,7 @@ on:
 jobs:
   release:
     name: Create Github Release
-    runs-on: ${{ inputs.runs-on }}
+    runs-on: ${{ ((startsWith(inputs.runs-on, '[') || startsWith(inputs.runs-on, '{') || startsWith(inputs.runs-on, '"')) && fromJSON(inputs.runs-on)) || inputs.runs-on }}
```

i'm wondering if it'd be possible to use [contains](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#contains)  instead

```suggestion
    runs-on: ${{ (contains('[{\"', inputs.runs-on[0]) && fromJSON(inputs.runs-on)) || inputs.runs-on }}
```